--- foobillard-3.0/src/ball.c~	2004-02-23 02:09:01.000000000 +0100
+++ foobillard-3.0/src/ball.c	2004-04-23 09:13:05.515746429 +0200
@@ -1820,15 +1820,15 @@ void draw_balls( BallsType balls, myvec 
                 draw_ball(&balls.ball[i],cam_pos,cam_FOV,win_width,1);
             } else if( options_cuberef && cuberef_binds!=0 ){
                 glBindTexture(GL_TEXTURE_CUBE_MAP_ARB, cuberef_binds[i]);
-#ifdef GL_VERTEX_PROGRAM_NV
-                {
-                    myvec cam_pos2;
+//#ifdef GL_VERTEX_PROGRAM_NV
+//                {
+//                    myvec cam_pos2;
 //                    cam_pos2=vec_scale(vec_unit(vec_diff(cam_pos,balls.ball[i].r)),BALL_D/2.5);
-                    cam_pos2=balls.ball[i].r;
-                    glProgramParameter4fNV( GL_VERTEX_PROGRAM_NV, 10, /* c[10] */
-                                           BALL_D/2.0,BALL_D/2.5,0,0 );
-                }
-#endif
+//                    cam_pos2=balls.ball[i].r;
+//                    glProgramParameter4fNV( GL_VERTEX_PROGRAM_NV, 10, /* c[10] */
+//                                           BALL_D/2.0,BALL_D/2.5,0,0 );
+//                }
+//#endif
                 draw_ball(&balls.ball[i],cam_pos,cam_FOV,win_width,0);
             }else{
                 draw_ball(&balls.ball[i],cam_pos,cam_FOV,win_width,0);
